src/main/java/uk/gov/hmcts/reform/ccd/client/CoreCaseDataConfiguration.java

```diff
@@ -5,9 +5,19 @@
 import feign.jackson.JacksonDecoder;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Primary;
+import org.springframework.http.converter.json.Jackson2ObjectMapperBuilder;
 import uk.gov.hmcts.reform.ccd.client.model.SearchCriteria;
 
 class CoreCaseDataConfiguration {
+
+    @Bean
+    @Primary
```

Comment:
I don't think this is a good idea.
From what I remember this makes it difficult for other teams to override the object mapper bean.

It would be better to add the ignore unknown properties annotation to all the classes instead.

Reply:
ha... That's a good point.. I have removed this now..